database/interfaces.go

```diff
@@ -333,6 +334,7 @@ type ObjectUserAttributesDB interface {
 	) ([]*entry.ObjectUserAttribute, error)
 
 	GetObjectUserAttributesCount(ctx context.Context) (int64, error)
+	GetObjectUserAttributesCountByObjectID(ctx context.Context, objectID umid.UMID, attributeName string, sinceTime *time.Time) (int64, error)
```

Comment:
hmm, now that I see these in this PR: these should be `uint`, they can't be negative :)